infra/us-east-1/prod/rds/bastion/key-pair/terragrunt.hcl

```diff
@@ -0,0 +1,41 @@
+locals {
+
+  # Automatically load environment-level variables
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract out common variables for reuse
+  project        = local.account_vars.locals.project
+  app            = "marlowe-runtime"
+
+  bastion_name          = "${local.project}-${local.app}-bastion"
+  bastion_key_pair_name = "${local.project}-${local.app}-db-bastion-kp"
+
+  tags = {
+    organization = "iog"
+    tribe        = "smart-contracts"
+    environment  = "prod"
+    function     = "bastion-key-pair"
+    App          = "marlowe-runtime"
+    Environment  = "prod"
+    Terraform    = "true"
+    Project      = local.project
+    Resource     = local.bastion_name
+  }
```

Comment:
Please check if we can use the tags defined by the provider: 
https://github.com/input-output-hk/sc-dev-platform/blob/main/infra/terragrunt.hcl#L64